src/testdata/json_tests/interface_ipv4.json

```diff
@@ -0,0 +1,307 @@
+                                            "type": "integer",
+                                            "title": "The Prefix-length Schema",
+                                            "default": 0
+                                          }
+                                        }
+                                      },
+                                      "config": {
+                                        "$id": "#/properties/openconfig-interfaces:interface/items/properties/subinterfaces/properties/subinterface/items/properties/openconfig-if-ip:ipv4/properties/addresses/properties/address/items/properties/config",
+                                        "type": "object",
+                                        "title": "The Config Schema",
+                                        "required": [
+                                          "ip",
+                                          "prefix-length"
+                                        ],
+                                        "properties": {
+                                          "ip": {
+                                            "$id": "#/properties/openconfig-interfaces:interface/items/properties/subinterfaces/properties/subinterface/items/properties/openconfig-if-ip:ipv4/properties/addresses/properties/address/items/properties/config/properties/ip",
+                                            "type": "string",
+                                            "title": "The Ip Schema",
+                                            "default": "",
+                                            "pattern": "^(.*)$"
+                                          },
+                                          "prefix-length": {
+                                            "$id": "#/properties/openconfig-interfaces:interface/items/properties/subinterfaces/properties/subinterface/items/properties/openconfig-if-ip:ipv4/properties/addresses/properties/address/items/properties/config/properties/prefix-length",
+                                            "type": "integer",
+                                            "title": "The Prefix-length Schema",
+                                            "default": 0
+                                          }
+                                        }
+                                      }
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}
```

Comment:
Should you do a DELETE and GET at the end of this test to reset the interface config back to its original configuration?

Reply:
Okay added cleanup operation.